Run3Detector/analysis/utilities/milliqanProcessor.py

```diff
@@ -10,14 +12,96 @@
 
 class milliqanProcessor():
 
-    def __init__(self, filelist, branches, schedule=None, cuts=None, plotter=None, max_events=None):
+    def __init__(self, filelist, branches, schedule=None, cuts=None, plotter=None, max_events=None, runQualityOverride=False, qualityLevel="tight"):
+        self.qualityLevelString = qualityLevel
+        self.runQualityOverride = runQualityOverride
+        
+        #Converting the quality level to an integer
+        self.qualityDict = {"single_trigger": -1, "loose": 0, "medium": 1, "tight": 2}
```

Comment:
this should just go into a constants file. Maybe just create processorConstants.py and put this there